_posts/2018-11-12-cloud-cloud-computing.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: The cloud and cloud  computing
+categories: developer
+author: Melvin Atieno
+blog-image: cloud-computing/cloud.jpg
+intro: What is the cloud? Who made the cloud? Where is the cloud? How does it work? Why is it indestructible? What is cloud computing? What other forms of computing are there? These are among of the questions this blog intends to answer.
+
+---
+{:.post-figure}
+![The cloud](/assets/images/blog/{{page.blog-image}}){:class="img-responsive center"}
+
+
+{{page.intro}}
+
+The cloud is a pool of computer resources; servers, storage, databases, analytics, physical infrastructure and more, shared over the internet. The cloud is a virtual space that is accessible from anywhere around the globe. 
+
+The idea of the cloud is believed to have first been coined by [Joseph Carl Robnett Licklider](https://en.wikipedia.org/wiki/J._C._R._Licklider) in the 1960s.
+
+The cloud itself is indestructible because like we said earlier, it is a virtual space. The cloud is not physical.
+
+## Cloud computing.
+
+cloud computing is the delivery of cloud services.
+There are other types of computing such as [Distributed Computing](https://en.wikipedia.org/wiki/Distributed_computing),[Grid computing](https://en.wikipedia.org/wiki/Grid_computing),[Utility Computing](http://en.wikipedia.org/wiki/Utility_computing), and [Cluster Computing](http://en.wikipedia.org/wiki/Computer_cluster).
+
+Cloud services are delivered in a number of ways; Infrastructure as a service(IaaS), Platform as a service(PaaS) and Software as a service(SaaS). Here is a picture of resources you would traditionally manage.
+
+![Traditional IT model](/assets/images/blog/cloud-computing/traditional-IT.png){:class="img-responsive center"}
+
+### IaaS
+
+IaaS stands for Infrastructure as a service. With this service model, what you get, in simple terms, is a data center. Some aspects of computing are managed for  you by your service provider.
+
```

Comment:
Please mention VPS, as that is waht is used mainly in marketing IaaS

Reply:
Also add Kubernetes and containerisatio infrastructure and where that fits into the cloud. is it IaaS, PaaS etc. THen give examples of providers